src/components/Carousel/index.tsx

```diff
@@ -8,7 +8,7 @@ import Controller from './components/Controller'
 import styles from './index.module.scss'
 
 export interface SlideData {
-  title: string
```

Comment:
can we just remove this prop? is it being used anywhere in this component?

Reply:
Removed